### ISSUE TITLE
请问我在执行你readme.md中的指令时出现以下错误，我该如何解决？

### ISSUE THREAD

**Issue by songjinjian-1995** [songyouwei/ABSA-PyTorch#141]
usage: train_apc.py [-h] [--model_name MODEL_NAME] [--dataset DATASET]
                    [--optimizer OPTIMIZER] [--initializer INITIALIZER]
                    [--learning_rate LEARNING_RATE] [--dropout DROPOUT]
                    [--l2reg L2REG] [--num_epoch NUM_EPOCH]
                    [--batch_size BATCH_SIZE] [--log_step LOG_STEP]
                    [--logdir LOGDIR] [--embed_dim EMBED_DIM]
                    [--hidden_dim HIDDEN_DIM]
                    [--pretrained_bert_name PRETRAINED_BERT_NAME]
                    [--use_bert_spc USE_BERT_SPC]
                    [--use_dual_bert USE_DUAL_BERT]
                    [--max_seq_len MAX_SEQ_LEN]
                    [--polarities_dim POLARITIES_DIM] [--hops HOPS]
                    [--SRD SRD] [--hlcf {cascade,parallel}] [--lcf LCF]
                    [--lce {True,False}] [--lcp {True,False}] [--sigma SIGMA]
                    [--repeat REPEAT]
train_apc.py: error: unrecognized arguments: --config experiments_apc.json

**Comment by yangheng95** [comment]
这个库中没有train_apc.py，你可能说的是LC-ABSA中的代码，去掉--config 和后面的参数就可以运行了，我后面会修复这个问题

**Comment by songjinjian-1995** [comment]
但当我去掉后面部分时会出现OSError: file bert_pretrained_laptop not found的问题

**Comment by yangheng95** [comment]
README中提到了这是从BERT-ADA中下载的adapted BERT，如果你不打算下载，可以在experiments_apc.py中把bert_pretrained_laptop改成bert-base-uncased

**Comment by songjinjian-1995** [comment]
感谢您的回复
